Documentation/policies/package-servicing.md

```diff
@@ -0,0 +1,50 @@
+# .NET Package Servicing Model
+
+Most packages in the .NET ecosystem support only the latest version. Many packages from the .NET team operate on this same model. A subset of our packages operate on another model where multiple version bands are supported at the same time, matching in-support runtimes, offering more flexibility to users.
+
+Users should aim to maintain their package references to be as close to the supported version as possible. Being close to a supported version means that you have most or all security fixes and likely have no latent compatibility risk that can be present when there is a big gap between your reference and the supported version.
+
+## Latest version servicing
+
+The default package servicing model is “latest version”. The team publishes an updated package with a security and/or functionality fix. Users can upgrade to the new latest version to get the fix.
+
+The following libraries use this servicing model:
+
+- [.NET Aspire](https://github.com/dotnet/aspire)
+- [ML.NET](https://github.com/dotnet/machinelearning)
+- [Maintenance-packages](https://github.com/dotnet/maintenance-packages)

```

Comment:
It would be nice to add dotnet/extensions here as well.

One recurring point of confusion is that we have `Microsoft.Extensions.*` packages with different support models based on the repo it comes from. For MS.Ext libraries coming from runtime and aspnetcore, it uses Runtime-band servicing. For MS.Ext libraries in dotnet/extensions, it uses "Latest version servicing".